pandas/tests/indexes/test_interval.py

```diff
@@ -348,6 +348,109 @@ def test_get_loc_value(self):
         idx = IntervalIndex.from_arrays([0, 2], [1, 3])
         pytest.raises(KeyError, idx.get_loc, 1.5)
 
+    @pytest.mark.xfail(reason="new indexing tests for issue 16316")
+    def test_get_loc_value(self):
+
+        right = IntervalIndex.from_tuples([(0, 1), (2, 3)], closed='right')
+
+        pytest.raises(KeyError, right.get_loc(-0.5) )
```

Comment:
This doesn't work. You need to use one of these forms:
```
pytest.raises(KeyError, right.get_loc, -0.5)
```
or:
```
with pytest.raises(KeyError):
    right.get_loc(-0.5)
```

The second version is usually easier to read because you can use normal syntax, but in this case (with one argument) there might be something to be said for fitting on one line.